1-exercises/B-setting-values/exercise2.js

```diff
@@ -26,6 +27,9 @@ let student = {
 */
 
 // write code here
+if ((student["attendance"] = 90 && student["examScore"] > 60)) {
+  student["hasPassed"] = true;
+} else student["hasPassed"] = false;
 
```

Comment:
well done, also you can use ternary operator

Reply:
done:)